src/modules/villages/village_handler.ts

```diff
@@ -11,4 +12,17 @@ router.get(
     res.status(httpStatus.OK).json(await Service.findAllWithLocation(req.query))
```

Comment:
this is bad code, it should be defined first, into const

example ` 
const result: object | any = await Service.findAllWithLocation(req.query)`

or you can't set in interface

if you have already followed the existing base code, it should be replaced first before it becomes a lot of files

cc @ayocodingit

Reply:
jadi di `.json()` lebih baik di define terlebih dahulu

contoh const result = bla.bla nanti di `.json(result)`

pros nya apa : 
lebih enak di baca dan sesuai standar juga
cons : 
mungkin agak ribet harus di define terlebih dahulu, akan tetapi tidak sesuai standar TS

Reply:
iyups setuju setuju, thank mang @firmanJS , masih mencari mana yang bagus nya

Reply:
gas gas next pr naikin dulu fitur ini :runner: